.github/workflows/check_bazel_tests.yml

```diff
@@ -16,10 +16,6 @@ jobs:
     steps:
     - uses: actions/checkout@v4
 
-
-    - name: Search for duplicated error codes
```

Comment:
Why did you remove it?

Reply:
There is another workflow which runs the same check:
https://github.com/exasol/script-languages-release/blob/refactoring/950_refactor_options_line_parser/.github/workflows/check_error_codes.yml